src/hyperlight_host/src/hypervisor/gdb/hyp_debug.rs

```diff
@@ -479,3 +479,15 @@ pub mod kvm {
         }
     }
 }
+
```

Comment:
Can this be moved into an mshv specific file?

Reply:
Yes, I'll split them in two separate files.

Reply:
Moved to mshv specific different file